test/test_parse_api_display/api_display.test.cpp

```diff
@@ -0,0 +1,64 @@
+#include <unity.h>
+#include <bmp.h>
+#include <api_response_parsing.h>
+
+void assert_response_equal(ApiDisplayResponse expected, ApiDisplayResponse actual)
+{
+  TEST_ASSERT_EQUAL(expected.outcome, actual.outcome);
+  TEST_ASSERT_EQUAL_STRING(expected.image_url.c_str(), actual.image_url.c_str());
+  TEST_ASSERT_EQUAL(expected.update_firmware, actual.update_firmware);
+  TEST_ASSERT_EQUAL_STRING(expected.firmware_url.c_str(), actual.firmware_url.c_str());
+  TEST_ASSERT_EQUAL_UINT64(expected.refresh_rate, actual.refresh_rate);
+  TEST_ASSERT_EQUAL(expected.reset_firmware, actual.reset_firmware);
+  TEST_ASSERT_EQUAL(expected.special_function, actual.special_function);
+}
+
+void test_parseResponse_apiDisplay_success(void)
```

Comment:
Tests like this make it really clear what form of JSON the firmware is expecting, and a really convenient place to begin when incorporating future API changes.